src/main/java/com/points/node/binarytree/BinaryTreeBuilder.java

```diff
@@ -0,0 +1,58 @@
+package com.points.node.binarytree;
+
+import java.util.List;
+
+import com.points.node.twochild.Node;
+
+public class BinaryTreeBuilder {
```

Comment:
A 'builder' should probably build(). Then the interface should be pulled out (implements IBuilder).